storybookjs/storybook
# Addon-docs: Rich props table UI

Issue: https://github.com/storybookjs/storybook/issues/8828

## What I did

Add styling for the new rich props table.

Props to @fraincs for making this happen!

## How to test

- Is this testable with Jest or Chromatic screenshots? No
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

Can be test in the cra-ts-kichen-sink: `?path=/docs/docgen-types--page`

![image](https://user-images.githubusercontent.com/794579/69177983-a4a86580-0ad6-11ea-8810-d6d356779b03.png)


If your answer is yes to any of these, please make sure to include it in your PR.
